tools/explorer/pkg/escrow/stellar.go

```diff
@@ -352,11 +353,19 @@ func (e *Stellar) payoutFarmers(id schema.ID) error {
 			continue
 		}
 
-		destination, err := addressByAsset(farm.WalletAddresses, config.Config.Asset)
-		if err != nil {
-			// FIXME: this is probably not ok, what do we do in this case ?
-			log.Error().Err(err).Msgf("failed to find address for %s for farmer %d", config.Config.Asset, farm.ID)
-			continue
+		// default use freeTFT issuer as destination
+		// if a reservation is free to use we send the tokens
+		// back to the issuer, this will freeze the tokens from being used again
+		destination := e.wallet.GetFreeTFTIssuer()
+
+		// if the reservation is not free we assume the currency is TFT
+		if !rpi.Free {
```

Comment:
I would prepare for the future and so something like
```go
if rpi.Free {
	destination := e.wallet.GetFreeTFTIssuer()
} else {
	destination, err = getAddressFarmer(farm.WalletAddresses, config.Config.Asset)
	if err != nil {
		// FIXME: this is probably not ok, what do we do in this case ?
		log.Error().Err(err).Msgf("failed to find address for %s for farmer %d", config.Config.Asset, farm.ID)
		continue
	}
}
```
So this code will work even if we use any other asset